### PR TITLE
AA-8250: update the flow source create and add the endpoint to the resouce

### PR DESCRIPTION
**Pull request by atilley1**

### CODE REVIEW

File: src/logs/flow-source-create-request.struct.apib [hunk 1]
```diff
@@ -1,16 +1,13 @@
 ## FlowSourceCreateRequest (object)
 
-+ flowSourceId (Guid)
-    The unique identifier of the flow source.
-
-+ orgId (number)
-    The unique identifier of organization with which this flow source will be associated.
++ siemFlowSourceId (number)
+    The flow source in qradar.
```
Comment (straineanu) [inline src/logs/flow-source-create-request.struct.apib]: Use the descriptions from https://github.com/armor/api-docs/blob/dev/src/logs/flow-source-edit-request.struct.apib as much as possible.

File: src/logs/log-management.resource.apib [hunk 2]
```diff
@@ -172,6 +172,14 @@ have the necessary permissions.
 + Request (application/json)
 + Response 200 (application/json) (array[FlowSource])
 
+### Create Flow Source [POST]
+Adds a flow source to the pool of available flow sources.
+
+**This is an internal resource and only accessible to users with a specific claim on their
```
Comment (straineanu) [inline src/logs/log-management.resource.apib]:
```suggestion
**🔒 This is an internal resource and only accessible to users with a specific claim on their
```

File: src/logs/log-management.resource.apib [hunk 3]
```diff
@@ -172,6 +172,14 @@ have the necessary permissions.
...
+authorization token. If this claim is missing, the API will return a 403 response.**
+
++ Request (application/json) (FlowSourceCreateRequest)
++ Response 200 (application/json) (FlowSource)
```
Comment (straineanu) [inline src/logs/log-management.resource.apib]:
```suggestion
+ Response 200 (application/json) (FlowSource)
+ Response 400 (application/json) (Message)
+ Response 403 (application/json) (Message)
```

File: src/logs/flow-source-create-request.struct.apib (hunk 1 above)
Comment (armorpreston) [reply src/logs/flow-source-create-request.struct.apib]: I'm doing it

**Pull request merged**